src/main/java/io/github/aquerr/steamwebapiclient/response/TradeOfferResponse.java

```diff
@@ -0,0 +1,159 @@
+package io.github.aquerr.steamwebapiclient.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class representing the result of <a href="https://partner.steamgames.com/doc/webapi/IEconService#GetTradeOffer">https://partner.steamgames.com/doc/webapi/IEconService#GetTradeOffer</a>
+ */
+@Data
+public class TradeOfferResponse implements SteamWebApiResponse {
+
+    @JsonProperty("response")
+    private TradeOfferWrapper response;
+
+    @Data
+    public static class TradeOfferWrapper {
+
+
+        @JsonProperty("offer")
+        private TradeOffer offer;
+
+        @Data
+        public static class TradeOffer {
+
+            /**
+             * The trade offer id.
+             */
+            @JsonProperty("tradeofferid")
+            private String tradeOfferId;
+
+            /**
+             * The account id of other user.
+             */
+            @JsonProperty("accountid_other")
+            private Long otherUserAccountId;
+
+            /**
+             * Message of the trade.
+             */
+            @JsonProperty("message")
+            private String message;
+
+            /**
+             * Trade offer expiration time in seconds.
+             */
+            @JsonProperty("expiration_time")
+            private long expirationTimeSeconds;
+
+            /**
+             * Trade offer state.
+             */
+            @JsonProperty("trade_offer_state")
+            private int state;
```

Comment:
Can you add an enum here? https://developer.valvesoftware.com/wiki/Steam_Web_API/IEconService#ETradeOfferState